### DISCUSSION TITLE
Latest tenant is not returning on login again

### DISCUSSION THREAD

**Discussion by fagnerdireito** [filamentphp/filament#7528] [General]
Hey, so I followed the docs' recommendations to create a multi-tenancy app, but here's the thing: I set up the default tenant, but whenever I switch tenants and click on the app's logo, it goes back to the previous tenant. Same thing happens when I logout and login again, and even when I navigate to Filament's default pages, like the profile page. It's kinda frustrating, you know? Any ideas on how to fix this?

    <?pho       
 
    namespace App\Models;
 
    use Filament\Models\Contracts\FilamentUser;
    use Filament\Models\Contracts\HasDefaultTenant;
    use Filament\Models\Contracts\HasTenants;
    use Illuminate\Database\Eloquent\Model;
    use Illuminate\Database\Eloquent\Relations\BelongsTo;
 
    class User extends Model implements FilamentUser, HasDefaultTenant, HasTenants
    {
    
    ...
    	// default tenant
		public function getDefaultTenant(Panel $panel): ?Team
		{
			return $this->latestTeam;
		}

		public function latestTeam(): BelongsTo
		{
			return $this->belongsTo(Team::class, 'latest_team_id');
		}
    }

**Comment by danharrin** [comment]
Sounds like a bug, and when clicking on the app logo it should send you to the homepage of the current tenant

**Reply by fagnerdireito** [reply]
yes it seems to be a bug, in addition to clicking on the logo to return the previous tenant, this also happens when navigating to other pages such as login and profile page.

**Reply by danharrin** [reply]
Please open an issue then with a reproduction repository
